packages/react-ui/storybook/.storybook/preview.js

```diff
@@ -48,7 +48,9 @@ export const parameters = {
         ['Palette', 'Typography'],
         'Common',
         'Widgets',
-        ['WrapperWidgetUI', 'FormulaWidgetUI', 'CategoryWidgetUI']
+        ['FormulaWidget', 'CategoryWidget', 'HistogramWidget', 'PieWidget'],
+        'WidgetsUI',
+        ['CategoryWidgetUI']
```

Comment:
@AdriSolid aren't we losing here the other UI widgets?

Reply:
Nop, that's actually the order, let me add same order as `Widget`s :+1: